app/assets/stylesheets/forever_style_guide/modules/_nav-dropdowns.scss

```diff
@@ -68,6 +77,10 @@ $nav-icon-width: 60px;
     @include active-style($color-accent);
   }
 }
+//ugly cousin active state
```

Comment:
maybe something like 
- a mixin that takes a color and sets active style and color
- one class for now .navbar-emphasized and we assume navbar-emphasized defaults to orange
- in the future a class could be added like navbar-emphasized-primary and that would be styled w/ the mixin? 

Something like that? but probably not a need to do a full like @each programattic development of an emphasized style in every palatte color at this time.